packages/wrapped-keys/src/lib/generated/litActions/common/exportPrivateKey.js

```diff
@@ -0,0 +1,2 @@
+// DO NOT EDIT THIS FILE. IT IS GENERATED ON BUILD. RUN `yarn bundle` IN THE ROOT DIRECTORY TO UPDATE THIS FILE.
+module.exports = "";
```

Comment:
These are dummy files. The lit actions build step will populate all these with the actual lit action code that is sent to the nodes, instead of the IPFS CID

In the bundle without the lit actions code this will not satisfy the `if (!litActionCode) {` condition and therefore the IPFS CID will be used